api/appendix_extensions.asciidoc

```diff
@@ -26,21 +26,18 @@ Within each group, extensions are listed in alphabetical order by their
 names.
 
 
-[[boilerplate-provisional-header]]
-== Provisional Extensions
+[[boilerplate-experimental-header]]
+== Experimental Extensions
 
-_Provisional_ OpenCL extensions described in this appendix have been
-Ratified under the Khronos Intellectual Property Framework.
-They are being made publicly available as provisional extensions to enable
-review and feedback from the community.
-While an extension is provisional, features may be added, removed, or
-changed in non-backward compatible ways.
+_Experimental_ OpenCL extensions described in this appendix are being made
+publicly available as experimental extensions to enable review and feedback
+from the community. While an extension is experimental, features may be
```

Comment:
This sentence reads a little strangely now, since it uses "experimental" twice.  How about:

```suggestion
_Experimental_ OpenCL extensions described in this appendix are being made
publicly available to enable review and feedback
from the community. While an extension is experimental, features may be
```